tests/pytests/integration/runners/test_nacl.py

```diff
@@ -0,0 +1,170 @@
+"""
+Tests for the nacl runner
+"""
+
+import pytest
+
+import salt.config
+import salt.utils.stringutils
+from tests.support.mock import patch
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+]
+
+try:
+    import salt.runners.nacl as nacl
+
+    HAS_PYNACL = True
+except (ImportError, OSError, AttributeError):
+    HAS_PYNACL = False
```

Comment:
Do me a favor, apply this patch locally, and try to run every nacl test you just added
```diff
diff --git a/noxfile.py b/noxfile.py
index 38e5b18585..e30e349265 100644
--- a/noxfile.py
+++ b/noxfile.py
@@ -354,6 +354,14 @@ def _install_requirements(
         install_command += EXTRA_REQUIREMENTS_INSTALL.split()
         session.install(*install_command, silent=PIP_INSTALL_SILENT)

+    session_run_always(
+        session,
+        "pip",
+        "uninstall",
+        "-y",
+        "pynacl"
+    )
```

Then you'll see why those `importorskip` are a **must**